Semana11/Ejercicio1.py

```diff
@@ -0,0 +1,14 @@
+class Circle:
+    def __init__(self, radio = 0):
+        self.radio = radio
+
+    def get_area(self):
+        area= 3.14159 * self.radio**2
+        return area
+
+
+my_circle = Circle()
+radio = float(input("Ingrese el radio..."))
```

Comment:
Move this line before line 10 so that you pass the radio when creating the Circle and make use of the constructor